util/delete_roles.sh

```diff
@@ -38,16 +33,19 @@ while (( "$#" )); do
 done
 
 # Check if required arguments are provided
-if [ -z "$RG" ] || [ -z "$LOCATION" ]; then
-    echo "Please ensure that --resource-group and --location are both provided" >&2
+if [ -z "$RG" ] ; then
+    echo "Please ensure that --resource-group is provided" >&2
     HELP=1
 fi
 
 if [ $HELP == 1 ]; then
-    echo Usage: delete_roles.sh --resource-group RG --location LOCATION [--delete-resource-group] 1>&2
+    echo Usage: delete_roles.sh --resource-group RG [--delete-resource-group] 1>&2
     exit 1
 fi
 
+LOCATION=$(az group show -n $RG --query location -o tsv)
+LOCATION=${LOCATION::-1} #remove trailing newline character
```

Comment:
interesting - you may want to check if this works on ubuntu, as I know Ubuntu uses dash instead of bash, so some more advanced bash features may not exist.